File: js/IntersectionOfTwoArrays.js


var intersection = function(nums1, nums2) {
    var ret = [];
    for(let i = 0; i < nums1.length; i++){
        if((nums2.includes(nums1[i])) && !(ret.includes(nums1[i]))){
            ret.push(nums1[i]);
        }
    }

    return ret;
};

File: csharp/IntersectionOfTwoArrays.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LeetCode
{
    class IntersectionOfTwoArrays
    {
        public int[] Intersection(int[] nums1, int[] nums2)
        {
            return nums1.Intersect(nums2).ToArray();
        }
    }
}
